assets/js/base/components/product-list/index.js

```diff
@@ -124,12 +124,11 @@ const ProductList = ( {
 	};
 
 	const getClassnames = () => {
-		const { columns, rows, className, alignButtons, align } = attributes;
+		const { columns, rows, alignButtons, align } = attributes;
 		const alignClass = typeof align !== 'undefined' ? 'align' + align : '';
 
 		return classnames(
 			layoutStyleClassPrefix,
-			className,
```

Comment:
If this is intentional, should the `product-list/container.js` have the custom`attributes.className` then?

Reply:
That's related to the comment above. :point_up: We are always printing block classNames in `edit.js` and the `save` function, but we must not print them in inner components, otherwise the same class is applied to the wrapping element and an inner one.

In this case, `/all-products/edit.js` and `/all-products/index.js` already print the attributes className in the wrapping `<div>`, so we shouldn't print it again in the `<ProductList>` component.